.github/workflows/luarocks-build.yml

```diff
@@ -65,7 +65,7 @@ jobs:
         run: |
           cd $GITHUB_WORKSPACE/tarantool/build
           cmake --build . --config $BUILD_TYPE --target install -- -j
-          echo "::set-env name=PATH::$GITHUB_WORKSPACE/share/bin:$PATH"
+          echo "PATH=$GITHUB_WORKSPACE/share/bin:$PATH" >> $GITHUB_ENV
```

Comment:
Don't know why extra path is needed, but I don't mind the change.

Reply:
Yes, I suggest fix CI in this way. Not sure we should rebuild Tarantool every time. One day we can change it to [setup-tarantool](https://github.com/tarantool/setup-tarantool)

Reply:
setup-tarantool is useful for using stable releases, due to caching involved. Here we need to compile against master, where caching would not be useful that much.

Reply:
However after https://github.com/tarantool/setup-tarantool/issues/9 it'll able to install latest master from a package. It should be faster than cloning and building tarantool.